app/models/methodologies.rb

```diff
@@ -1,11 +1,10 @@
 class Methodologies
   NAME_VALUES = {
-    interview:   'Interview',
-    usability:   'Usability testing',
-    survey:      'Survey',
-    focusgroup:  'Focus group',
-    codesign:    'Codesign workshop',
-    observation: 'Observation/field study',
+    interview:   'a one-on-one interview',
+    usability:   'an observation of trying out (the new tool) we’re designing',
```

Comment:
Please remove the brackets. Expected outcome:
> an observation of trying out the new tool we’re designing

Reply:
Can we safely assume it will always be a new tool?